test/robust/compare/output1__Stub.js

```diff
@@ -35,12 +63,36 @@ describe('/', function() {
           return;
         }
 
-        body.should.have.property('name');
+        /*eslint-disable*/
+        validator.validate(body, schema).should.be.true;
```

Comment:
Now that schema is stored in `schema`, this line doesn't have to be ignored by ESHint. Or maybe I'm missing something?